test/e2e/performanceprofile/functests/5_latency_testing/latency_testing.go

```diff
@@ -96,7 +96,7 @@ var _ = table.DescribeTable("Test latency measurement tools tests", func(testGro
 		clearEnv()
 		testDescription := setEnvAndGetDescription(test)
 		By(testDescription)
-		output, err := exec.Command(testExecutablePath, "-ginkgo.focus", test.toolToTest).Output()
+		output, err := exec.Command(testExecutablePath, "-ginkgo.v", "-ginkgo.focus", test.toolToTest).Output()
```

Comment:
This is for us to be able to read the executable output regardless of the test results (whether it passed or failed).